cli/command/stack/kubernetes/services.go

```diff
@@ -140,3 +133,27 @@ func RunServices(dockerCli *KubeCli, opts options.Services) error {
 	}
 	return formatter.ServiceListWrite(servicesCtx, services, info)
 }
+
+func filterServicesByName(services []swarm.Service, f filters.Args, stackName string) []swarm.Service {
+	names := f.Get("name")
```

Comment:
Would it be clearer to pass `names []string` as argument instead of the filter-set?